earthpy/tests/test_mask.py

```diff
@@ -28,16 +28,26 @@ def test_masked_arr_returned():
     masked = make_apply_mask(im, im_mask, [0,4])
     assert np.ma.is_masked(masked)
 
-# def test_val_list_provided:
-#     """Tuple works just fine. what about some other object type? like? """
-#
-#     with pytest.raises(AttributeError):
-#         make_apply_mask(im,
-#                     mask_arr=im_mask,
-#                     vals=[0,4])
+def test_val_list_provided():
+    """Tuple works just fine. what about some other object type? like?
+        how about a singular value? if not that, maybe a dictionary?"""
+
+    with pytest.raises(AttributeError):
+        make_apply_mask(im,
+                    mask_arr=im_mask,
+                    vals=0)
 
 # Test that a mask is provided to apply mask that doesn't have 1's in it.
 # should return - there's nothing to mask here!
+def test_mask_provided_with_no_ones():
```

Comment:
Can you kindly turn the comments into docstrings that describe what the test does @joemcglinchy  ?? this is really great!!